osmorc/osmorc-jps-plugin/src/org/jetbrains/osgi/jps/build/BndWrapper.java

```diff
@@ -194,6 +196,45 @@ public void build(@NotNull Map<String, String> properties, File @NotNull [] clas
     }
   }
 
+  public void build(@Nullable File bndFile,
+                    @NotNull Map<String, String> properties,
+                    @NotNull Properties mavenProperties,
+                    File @NotNull [] classPath,
+                    File @NotNull [] srcPath,
+                    @Nullable File base,
+                    @NotNull File moduleOutputDir,
+                    @NotNull File outputFile) throws Exception {
+    try (Builder builder = new ReportingBuilder(myReporter, new Processor(mavenProperties, false))) {
+      if (LOG.isDebugEnabled()) {
+        builder.setTrace(true);
+        LOG.debug("OSGi bundle bnd file: " + bndFile + ", properties: " + properties + ", Maven properties: " + mavenProperties);
```

Comment:
`mavenProperties` is used only for logging - is that correct?

Reply:
They're passed to the constructor of `Processor` on line 207.